virttest/utils_misc.py

```diff
@@ -2739,31 +2738,49 @@ def format_windows_disk(session, did, mountpoint=None, size=None, fstype="ntfs")
     return False
 
 
-def format_linux_disk(session, did, mountpoint, size, fstype="ext3"):
+def format_linux_disk(session, did, mountpoint=None, size=None, fstype="ext3"):
     """
     Create a partition on disk in linux guest and format and mount it.
 
     :param session: session object to guest.
     :param did: disk serial, kname or wwn.
     :param mountpoint: mount point for the disk.
     :param fstype: filesystem type for the disk.
-    :param ostype: guest os type 'windows' or 'linux'.
     :return Boolean: disk usable or not.
     """
-    list_disk_cmd = "lsblk -p -oKNAME,MOUNTPOINT,SERIAL,SIZE,MODEL,WWN"
+    list_disk_cmd = "lsblk -oKNAME,MOUNTPOINT,SIZE,MODEL"
```

Comment:
def list_guest_volumes(partition=False):
      cmd = "ls /dev/[vhs]d*"
      if not partition:
          cmd = "%s | grep -v [0-9]$" % cmd
      output, status = get_guest_cmd_status_output(vm, cmd)
      if status != 0:
          raise exceptions.TestFaild()
      return output.splitlines()